inginious/frontend/static/css/bootstrap-rtl.min.css

```diff
@@ -0,0 +1,14 @@
+/*!
+ * Bootswatch v4.5.3
```

Comment:
It's probably useful to make both the Bootstrap versions (LTR and RTL) match to minimize unwanted behaviours and benefit from the same feature set. The other one is still v4.1. Both the versions do not, for instance, use the exact same color palette in tabulars.